### DISCUSSION TITLE
General questions

### DISCUSSION THREAD

**Discussion by Mahmoud-87** [TDRA-ae/aegov-dls#15] [Q&A]
Is it acceptable that the container width in Figma appears nearly full-width, whereas on large screens it seems to be around 80%?
 
What happens to the FAQ category on mobile when there are numerous categories?
 
Should the bookmark remain active when users revisit the page if they've already added it to their bookmarks?
 
Is it mandatory for all static pages to adhere to the layout of the privacy policy page, which includes using the page navigation on the right sidebar?
 
The pagination is located on the right in the news listing but centered in services. Is it necessary to keep this distinction, or can we center it on all pages?

**Comment by arslanbutt9089** [comment]
I'll address each question individually, incorporating insights from best practices and design principles:

1. Container Width:

Acceptability depends on design goals and visual hierarchy. Consider:
Scannability: Does near-full-width enhance or hinder content readability on large screens?
Visual balance: Does 80% width create a more pleasing aesthetic and guide focus?
Breakpoints: Ensure responsiveness across devices for optimal viewing experiences.
2. FAQ Category on Mobile:

Recommendations:
Accordion or expandable menu for limited screen space.
Scrollable list with clear category headings for many categories.
Search functionality for quick navigation.
3. Persistent Bookmark:

General practice: Maintain bookmarked state for user convenience.
Exceptions: Reassess if content updates frequently or requires fresh engagement.
4. Static Page Layout:

Consistency crucial for usability:
Adhere to privacy policy layout for familiarity and predictability.
Consider exceptions if specific pages have unique content requirements.
5. Pagination Placement:

Consistency generally favored: Centered pagination enhances discoverability and aligns with common user expectations.
Exceptions: Right-aligned placement might suit specific content structures or layout constraints.
Additional Considerations:

User testing: Gather feedback to validate design choices and ensure a seamless user experience across devices.
Design system: Establish clear guidelines for consistency and maintainability.
Accessibility: Ensure designs cater to diverse needs and assistive technologies.
I'm ready to provide further guidance based on specific design requirements or context.
